guides/micronaut-graalvm-native-image-google-cloud-platform-cloud-run/micronaut-graalvm-native-image-google-cloud-platform-cloud-run.adoc

```diff
@@ -193,7 +193,10 @@ Service [micronautguide] revision [micronautguide-00002-fat] has been deployed
 [source, bash]
 ----
 curl -i https://micronautguide-li3tercjmq-ue.a.run.app/hello
-HTTP/2 200 
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks